snooty/rstspec.toml

```diff
@@ -1029,6 +1029,10 @@ type = "text"
 help = """Show a file path."""
 type = "text"
 
+[role.card-ref]
```

Comment:
Could we:
* Place this in the `mongodb` domain
* Add a help string

What directive will create the targets to be linked to?

Reply:
@i80and Since she's inheriting ref, will this not work to link to refs?

Reply:
@schmalliso well, it'll link to labels specifically Just making sure that's intended -- this is a new use case to me :D

Reply:
Reasonable! Good, I've not led her completely astray then (that is the desired use-case).

Reply:
@i80and I believe there is a help string already, unless you're referring to something else?

Reply:
@mango-db oh snap you're right -- sorry, I don't know how I missed that :D If we could be slightly more specific about what we mean by "styled reference" and what the intent is, that would be ideal. A help string should be enough for a writer to know when they might want to use it.